src/main.c

```diff
@@ -1,18 +1,25 @@
+#include <stdio.h>

```

Comment:
Wherer is stdio.h used? Usually it's used for printing to the console and wew don't have a console here.